user/plan-sharing.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Plan Sharing Guide
+layout: en
+
+---
+
+Assembla’s billing model allows customers to pay once for an entire portfolio of projects and allocate the allowance across the projects. Travis CI modified the billing model to map Assembla’s spaces to Travis CI Organizations and fulfill customer needs.  
+
+Travis CI introduces a flexible new feature that allows users to share paid plans across Assembla’s spaces. This innovation introduces two new roles to Travis CI plans: the Donor and the Receiver, providing users with more control over their resources. 
+
```

Comment:
We use generic descriptions below, however this feature will be active for Assembla users signing in to Travis. Shouldn't there be an explicit remark about it before moving to the description, like in blockquote section?